### DISCUSSION TITLE
Assertion 'nlist_a[axis_idx[dd]] >= 0' failed.

### DISCUSSION THREAD

**Discussion by Yi-FanLi** [deepmodeling/deepmd-kit#804] [Q&A] [answered]
I am testing our recently developed lammps PIMD program with DeepMD-kit v1.3.3-33-g816b803 version on a 192-atom water system with 1 bead for an NVE run. However, after running for about 150 steps, the simulation crushed, showing the following error message: 

lmp_mpi: /scratch/gpfs/yifanl/Softwares/deepmd-kit/source/op/descrpt.cc:451: void DescrptOp<Device, FPTYPE>::make_axis(std::vector<int>&, std::vector<int>&, c    onst int&, const std::vector<int>&, int, const std::vector<int>&, const std::vector<int>&, const std::vector<double>&, const SimulationRegion<double>&, bool)     const [with Device = Eigen::ThreadPoolDevice; FPTYPE = double]: Assertion `nlist_a[axis_idx[dd]] >= 0' failed.

The same error does not error when using "fix nve", which should be equivalent to the 1-bead NVE pimd run. Thus, there must be something wrong with the currently developing PIMD program.

My question is, what will trigger this error message in DeepMD-kit? What does the assertion 'nlist_a[axis_idx[dd]] >= 0' require?
<img width="1753" alt="Screen Shot 2021-06-27 at 8 54 17 PM" src="https://user-images.githubusercontent.com/39401945/123545332-0ab49e80-d78a-11eb-81f8-949a817cea3e.png">

**Comment by njzjz** [comment]
Could you please provide the model and trajectory of the last step?

**Reply by Yi-FanLi** [reply]
I was using the wrong mode. Now I froze a correct, smooth-version model and it works. Thanks!

**Comment by amcadmus** [comment, marked as answer]
You are using the non-smooth version of Deep Potential. It does not make sense to use it with NVE, because it does not conserve the total energy.

**Reply by Yi-FanLi** [reply]
I am using type="se_a" for the descriptor in the json script. So I think I am using the smooth version.

**Reply by Yi-FanLi** [reply]
I was using the wrong mode. Now I froze a correct, smooth-version model and it works. Thanks!
